Docker-compose.yml

```
@@ -0,0 +1,27 @@
+version: '3'
+services: 
+  app:
+    build: .
+    ports:
+      - "5000:5000"
+    command: bundle exec rails s -p 5000
+    depends_on:
+      - iis-sandbox
+      - v2-to-fhir
+    links:
+      - iis-sandbox
+      - v2-to-fhir
+    environment:
+      IIS_SANDBOX_USERNAME: mitre
+      IIS_SANDBOX_PASSWORD: mitre
+      IIS_SANDBOX_FACILITY_ID: MITRE Healthcare
+      IIS_SANDBOX_HOST: http://localhost:8081
+      V2_TO_FHIR_HOST: http://localhost:3000
```

```suggestion
      IIS_SANDBOX_HOST: http://iis-sandbox:8080
      V2_TO_FHIR_HOST: http://v2-to-fhir:3000
```

Prefer using docker container hostname